Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -453,6 +489,18 @@ void EnhancementsWidget::AddDescriptions()
       "scaling filter selected by the game.<br><br>Any option except 'Default' will alter the look "
       "of the game's textures and might cause issues in a small number of "
       "games.<br><br><dolphin_emphasis>If unsure, select 'Default'.</dolphin_emphasis>");
+  static const char TR_OUTPUT_RESAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Affects how the game output image is upscaled or downscaled to the window resolution.<br>"
+      "\"Default\" will rely on the GPU internal bilinear sampler which isn't gamma corrected."
+      "<br>\"Bilinear\" (gamma corrected) is a good compromise between quality and performance."
+      "<br>\"Bicubic\" is smoother than \"Bilinear\"."
+      "<br>\"Hermite\" might offer the best quality when upscaling,"
+      " at a slightly bigger perform cost.<br>\"Catmull-Rom\" is best for downscaling."
```

Comment:
`"<br>\"Hermite\" might offer the best quality when upscaling,"
      " at a slightly bigger perform cost.<br>\"Catmull-Rom\" is best for downscaling."`

Seems you meant to put Catmull-Rom on a new line?

Also, I'd suggest "may" rather than "might" here. Though honestly in general I'd prefer to avoid indefinites in the language of these tooltips but I can't actually test it right now so, whatever I guess.

Reply:
Sam has already changed all these tooltips for a follow up PR:
https://github.com/dolphin-emu/dolphin/pull/12018

I purposely used indefinites because it's subjective and it depends on a case by case basis. I didn't want people to say: "oh, it says it looks better thus I will use this."
My intention was for them to evaluate each themselves.

Reply:
Fair enough. Well, I'd still like for you to move from might to may though.